docs/tokensdk.md

```diff
@@ -0,0 +1,156 @@
+# The Fabric Token-SDK
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents** 
+
+- [Introduction](#introduction)
+- [Consumer Interaction Model](#consumer-interaction-model)
+- [Requirements and Use Cases](#requirements-and-use-cases)
+- [Prerequisites, Dependencies, Dependents, and Incompatibilities](#prerequisites-dependencies-dependents-and-incompatibilities)
+- [Terminology and Glossary](#terminology-and-glossary)
+- [Architecture, Interfaces, and Impact](#architecture-interfaces-and-impact)
+- [Developer Experience](#developer-experience)
+- [Command Line Interface (CLI)](#command-line-interface-cli)
+- [Performance, Scalability, and Resource Consumption](#performance-scalability-and-resource-consumption)
+- [Serviceability, Logging and Troubleshooting](#serviceability-logging-and-troubleshooting)
+- [Monitoring, Metrics and Events](#monitoring-metrics-and-events)
+- [High Availability and Disaster Recovery](#high-availability-and-disaster-recovery)
+- [Build, Packaging and Deployment](#build-packaging-and-deployment)
+- [Platform Support](#platform-support)
+- [Testing](#testing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+The Fabric Token-SDK (FTS, for short) is set of APIs and services for building token-based applications on Hyperledger Fabric, and potentially other platforms.
+
+The key Features are:
+* Uses the `Unspent Transaction Output` (UTXO) model for tracking token movements.
+* Manages cryptographic keys through `Wallets`, keeping track of owned unspent outputs.
+* Supports `various privacy levels`, from fully transparent to Zero-Knowledge Proofs for obfuscating transaction details.
+* Allows developers to create `custom services` on top of the core API for specific application needs.
+
+The Fabric Token SDK stack consists of several layers:
+* Services: Pre-built functionalities like assembling transactions and selecting unspent tokens.
+* Token API: Provides a common abstraction for interacting with tokens across different backends.
+* Driver API: Translates generic token operations into backend-specific calls (e.g., Fabric).
+* Drivers: Define token representation, operations, and validation rules for specific implementations.
+
+The SDK leverages the Fabric Smart Client stack for complex workflows, secure storage, and event listening.
+
+The Fabric Token SDK has evolved beyond its initial focus on Hyperledger Fabric.
```

Comment:
The sentence is unclear because it suggests the SDK expanded beyond Hyperledger Fabric but then mentions Fabric again, which is redundant. Consider rephrasing to clarify that the SDK now supports multiple platforms while still supporting Fabric. For example:

> Originally designed for Hyperledger Fabric, the SDK now supports other permissioned blockchains while continuing to work with Fabric.